src/validators.ts

```diff
@@ -18,16 +18,20 @@ let ORIGINAL_VALIDATOR_CONFIG: string | undefined;
 
 class ValidatorPkg {
   process?: ChildProcess;
-  validatorDir = resolve(_dirname, "..", "..", "validator");
+  validatorDir = "";
+  validatorCleanDir = resolve(_dirname, "..", "..", "validator", "clean");
+  validatorForkDir = resolve(_dirname, "..", "..", "validator", "fork");
 
-  start() {
+  start(fork?: boolean) {
     const binPath = getBinPath();
     if (!binPath) {
       throw new Error(
         `cannot start with: arch ${process.arch}, platform ${process.platform}`
       );
     }
 
+    this.validatorDir = fork ? this.validatorForkDir : this.validatorCleanDir;
```

Comment:
The Validator needs to know which config to use, based on if they are using a fork.
Worth noting that the `ValidatorDev` class, which is the docker based network for protocol development, cannot use a fork.  I don't see that as being useful, but if anyone on the core team finds themself wanting that I can set it up.